lib/hooks/useCustomers.ts

```diff
@@ -10,9 +10,9 @@ interface CustomerResponse {
 
 function useCustomers(): CustomerResponse {
     // mock endpoint
-    const endpoint = 'http://localhost:3000/api/customers';
+    // const endpoint = 'http://localhost:3000/api/customers';
     // actual endpoint
-    // const endpoint = process.env.NEXT_PUBLIC_API_URL + '/customer';
+    const endpoint = process.env.NEXT_PUBLIC_API_URL + '/customer';
```

Comment:
I'd use either concatenation or string templating throughout the project, not both here and there. Like in the NewProjectForm this could be 
```typescript
const endpoint = `${process.env.NEXT_PUBLIC_API_URL}/customer`;
```

Reply:
Fixed in bdfadda